### PR TITLE
Leaderboard list app: Add styling

### PR DESCRIPTION
**Pull request by Titus-Kiplagat**
- [x] No linter errors.
- [x] Used correct flow.
- [x] Documented the work.
- [x] Used best HTML & CSS practices.
- [x] Improved the look and feel of the app by adding styles.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -17,12 +19,84 @@ body {
 
 .heading {
   color: var(--heading-color);
+  font-size: 3rem;
```
Comment (tufoinnkuo10) [inline src/style.css]: - [optional] Great job following the wireframe design and making your style unique. Please, for user-friendliness, kindly make this circled colors more visible. I am sure you won't love if someone stress his/her eyes to read something caused by your design. 😄 nice work. 
Screenshot (1315)_LI

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:✔️✔️✔️✔️

Hi @Titus-Kiplagat ✔️✔️✔️✔️

I must say your project looks superb. Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍✔️✔️✔️✔️✔️✔️✔️
- Descriptive Pull Request✔️✔️✔️
- Professional Readme✔️✔️✔️
- Design on point✔️✔️✔️
- well deployed✔️✔️✔️
- wireframe well followed✔️✔️✔️✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 


tufoin


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Titus-Kiplagat) [reply src/style.css]: Yeah, that's absolutely true. Ensuring good text contrast is a fundamental aspect of web accessibility. Making sure that circled colors are more visible not only improves the overall user experience but also ensures that the design is inclusive and can be easily accessed by everyone, including those with visual impairments. Thanks for the feedback, and I'm glad you appreciate the effort to enhance user-friendliness! 😄

Comment (Titus-Kiplagat) [reply src/style.css]: Thank you for your positive feedback! I'm glad to hear you're happy with the project. If you think it's worth it, I would greatly appreciate a star for my effort. 🌟

**Pull request merged**